pkgs/by-name/ca/candy-icons/package.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, stdenv
```

```suggestion
, stdenvNoCC
```
NoCC since we're not compiling anything